helm/charts/hydra/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "v1.4.6"
+appVersion: "v1.7.4"
 description: A Helm chart for deploying ORY Hydra in Kubernetes
 name: hydra
-version: 0.0.1
+version: 0.0.2
```

This is changed automatically in the release process and should be reverted:

```suggestion
version: 0.0.1
```